extensions/build.gradle.kts

```diff
@@ -0,0 +1,119 @@
+import com.vanniktech.maven.publish.SonatypeHost
+import java.net.URI
+
+plugins {
+    id("java-library")
+    id("signing")
+    id("com.github.johnrengelman.shadow") version "7.1.1"
```

Comment:
didn't we keep all versions in `settings.gradle` or `gradle.properties`?

Reply:
I added a versions catalog and put it there